moveit_ros/move_group/src/move_group.cpp

```diff
@@ -309,7 +309,7 @@ int main(int argc, char** argv)
       RCLCPP_INFO(nh->get_logger(), "MoveGroup debug mode is OFF");
     }
 
-    rclcpp::executors::MultiThreadedExecutor executor;
```

Comment:
We have modified this part due to the fact that rmw_wait function of rmw_zenoh still has some problems. We must use SingleThreadedExecutor at the moment.